composer.json

```diff
@@ -11,7 +11,9 @@
     ],
     "require": {
         "php": ">=5.3",
```

Comment:
If so, i'd rewrite code to use "modern" features (such as short array syntax, $this in closures, etc). May we indeed refuse drop support v5.3.x?